src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag

```diff
@@ -4,24 +4,9 @@
 <c:set var="motdUrl" value="<%= Config.inst().STUDENT_MOTD_URL %>" />
 <c:if test="${not empty motdUrl}">
     <script>
-        var url = window.location.origin + '/' + '<c:out value="${motdUrl}" />';
-
-        $.ajax({
-            type: 'GET',
-            url: url,
-            success: function(data) {
-                $('#student-motd').html(data);
-            },
-            error: function(jqXHR, textStatus, errorThrown) {
-                console.log('AJAX request failed');
-            }
-        });
-
-        function closeMotd() {
-            $('#student-motd-container').hide();
-        }
+        var motdUrl = window.location.origin + '/<c:out value="${motdUrl}" />';
     </script>
-
+    <script type="text/javascript" src="/js/studentMotd.js"></script>
```

Comment:
Actually, the common practice is including it somewhere just before the page ends. This is because the DOM is rendered in a somewhat concurrent way with the JS execution and including JS way too early can be problematic (the targeted object may not be rendered yet). Unless, of course, the JS code is wrapped in `document.ready()`.